ml/src/main/scala/org/apache/spark/ml/h2o/algos/H2OAutoML.scala

```diff
@@ -43,6 +43,8 @@ class H2OAutoML(override val uid: String) extends Estimator[H2OMOJOModel]
 
   // Override default values
   setDefault(nfolds, 5)
+  logWarning("Default value of 'convertUnknownCategoricalLevelsToNa' parameter on H2OAutoML will be changed to 'false' from 'true' in the next" +
```

Comment:
I would remove 'on H2OAutoML'  and also avoid string concatenation.

Reply:
Thanks!